packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -69,11 +67,11 @@ class _TaskEntry<T> {
   }
   final TaskCallback<T> task;
   final int priority;
-  final String debugLabel;
-  final Flow flow;
+  final String? debugLabel;
+  final Flow? flow;
 
-  StackTrace debugStack;
-  Completer<T> completer;
+  late StackTrace debugStack;
+  late Completer<T> completer;
```

Comment:
Could these two be `late final`?

Reply:
Actually, it looks like completer could just be final and initialized right here instead of in the constructor?

Reply:
Indeed!